pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -522,6 +522,8 @@ type KubeletConfiguration struct {
 	// This flag, if set, instructs the kubelet to keep volumes from terminated pods mounted to the node.
 	// This can be useful for debugging volume related issues.
 	KeepTerminatedPodVolumes bool `json:"keepTerminatedPodVolumes,omitempty"`
+	// This flag, if set, disables use of a shared PID namespace for pods run by the docker CRI runtime.
+	DockerDisableSharedPID bool `json:"dockerDisableSharedPID,omitempty"`
```

Comment:
Just in case, make this a pointer (`*bool`) so that we can change the default it needed.